app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+    vaccination_issues = False
```

Comment:
There is no need for the `vaccination_issues` variable, you can return the required string at the moment of exception processing.